actionview/lib/action_view/helpers/tags/label.rb

```diff
@@ -75,7 +75,7 @@ def render_component(builder)
           end
 
           def skip_default_ids?
```

Comment:
It is used on line 53 (https://github.com/rails/rails/pull/29439/files#diff-6f3573047e15b0e2eba56e404d6a03f5R53). It uses the id to make the `for` attribute. It is no longer needed though so I'll remove it.